tvtk/pyface/actors.py

```diff
@@ -65,10 +65,10 @@ def cube_actor(center=(0, 0, 0), color=colors.blue, opacity=1.0):
     return actor
 
 
-def cylinder_actor(center=(0, 0, 0), radius=0.5, resolution=64,
+def cylinder_actor(center=(0, 0, 0), radius=0.5, height=1, heiresolution=64,
```

Comment:
Thanks, but there is a typo here that needs to be fixed.  The resolution has changed to heiresolution.